feed/json.njk

```diff
@@ -0,0 +1,38 @@
+---
+permalink: feed/feed.json
```

Comment:
I will make this change manually but you can re-use `permalink: "{{ metadata.jsonfeed.path | url }}"` here